demoproviders/barbican/java/org/openyolo/demoprovider/barbican/CredentialQualityScore.java

```diff
@@ -64,12 +64,13 @@ private static int getQualityScore(
         int score = 1;
 
         // email identifiers are recognisable and useful for ID token acquisition
-        if (CredentialClassifier.isEmailIdentifier(id)) {
+        if (AuthenticationMethods.EMAIL.toString().equals(authMethod)) {
             score <<= 1;
         }
 
         // prefer credentials that don't use passwords
-        if (!AuthenticationMethods.ID_AND_PASSWORD.equals(authMethod)) {
+        if (!AuthenticationMethods.PHONE.toString().equals(authMethod)
```

Comment:
I'll update the comment - what I'm really saying here is "prefer federated auth methods".